internal/runner-template.js

```diff
@@ -45,10 +47,20 @@ const options = {
   }
 };
 
+// Change to the bin directory so that plugins that emit files can do so
+// relative to the output root of the workspace.
+const cwd = process.cwd();
+const newDir = path.join(cwd, binDir);
+process.chdir(newDir);
+
 postcss(TEMPLATED_plugins)
     .process(cssString, options)
     .then(
         result => {
+          // Change back to the original cwd, because the .css and .css.map
+          // output paths were supplied relative to it.
```

Comment:
Rather than making sure we're in the right working directory at the right time, can you just adjust the output paths to be absolute before the first directory switch?

Reply:
Done.